docs/i18n/japanese/devops.md

```diff
@@ -863,100 +863,100 @@ pm2 save
 pm2 logs
 ```
 
-## Installing and Updating Azure Pipeline Agents
+## Azure Pipeline エージェントのインストールと更新
 
-See: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/v2-linux?view=azure-devops and follow the instructions to stop, remove and reinstall agents. Broadly you can follow the steps listed here.
+https://docs.microsoft.com/ja-JP/azure/devops/pipelines/agents/v2-linux?view=azure-devops を参照し、手順に従ってエージェントを停止、削除、再インストールします。 一般的には、ここに記載されている手順に従います。
```

Comment:
I changed this URL back to en-us on Crowdin. As far as we remember, we keep URLs to external websites to point to English page.